setup.py

```diff
@@ -17,10 +17,14 @@ def readme():
     url="https://github.com/plumerai/larq-zoo",
     packages=find_packages(),
     license="Apache 2.0",
-    install_requires=["numpy~=1.15", "larq~=0.8.1", "zookeeper~=0.5.3"],
+    install_requires=[
+        "numpy~=1.18",
+        "larq~=0.8.4",
+        "zookeeper @ git+git://github.com/AdamHillier/zookeeper@factories",
```

Comment:
This makes `pip install -e .` fail: 
```
Collecting zookeeper@ git+git://github.com/AdamHillier/zookeeper@factories
  Cloning git://github.com/AdamHillier/zookeeper (to revision factories) to /tmp/pip-install-pk5wh8lv/zookeeper
  Running command git clone -q git://github.com/AdamHillier/zookeeper /tmp/pip-install-pk5wh8lv/zookeeper
  WARNING: Did not find branch or tag 'factories', assuming revision or ref.
  Running command git checkout -q factories
  error: pathspec 'factories' did not match any file(s) known to git
ERROR: Command errored out with exit status 1: git checkout -q factories Check the logs for full command output.
```

Reply:
Ah yes sorry that'll be because I deleted the branch after it was merged into Zookeeper master. Waiting on one last thing before I do the Zookeeper release and can update the dependency here.

Reply:
In the meantime I'll point this to master.

Reply:
Now updated to the just-released new Zookeeper version.